src/components/BookList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import IndividualBook from './IndividualBook';
+import BookForm from './BookForm';
+import Navigation from './Navigation';
+
+const books = [
+  {
+    title: "The Handmaid's Tale",
+    author: 'Suzanne Collins',
+    category: 'Fiction',
+    completed: 81,
+    currentChapter: 17,
+  },
+  {
+    title: 'Great Expectations',
+    author: 'Suzanne Collins',
+    category: 'Classics',
+    completed: 71,
+    currentChapter: 12,
+  },
+];
+const BookList = () => (
+  <div>
+    <Navigation />
```

Comment:
- [ ] The Navigation bar should be present in the `App` component so it's visible on all the pages, kindly consider fixing this so your app behaves as expected

Reply:
- [ ] According to the previous reviewer's suggestion, please make sure to move your `<Navigation />` component to your App component, or consider creating a [`Layout Route`](https://reactrouter.com/en/main/start/concepts#layout-routes). This is important because ideally, you want your navigation bar to be visible to all pages of your app. Please consider implementing this change! 🤓 🔧